File: c11_svg/A_12_Emil_Lozev.rb
def circle x,y,r	
	'<circle cx="' + x.to_s + '" cy="' + y.to_s + '" r="' + r.to_s + '" stroke="black" stroke-width="3" fill="purple" />'
end

File.open("result.svg","w") do |f|
	f.write('<svg xmlns="http://www.w3.org/2000/svg">')
	f.write('<rect width="55" height="55" x="95" y="95" style="fill:rgb(0,0,255);stroke-width:3;stroke:rgb(0,0,0)"> </rect>')
	#f.write('<circle cx="170" cy="170" r="40" stroke="black" stroke-width="3" fill="purple" />')
	#f.write(circle(170,350,30))
	f.write(circle(67,105,30))	
	f.write(circle(120,70,30))
	f.write(circle(173,100,30))
	f.write(circle(167,160,30))	
	f.write(circle(125,173,30))
	f.write(circle(78,165,30))
	f.write( '<text x="100" y="250" fill="red">Emil Lozev</text>')
	f.write('</svg>')
end

File: c21/A_12_Emil_Lozev.js
var myFunction = function(var a[][]) {
    
}
rect(10, 10, 100, 100);
rect(200, 200, 50, 100);
fill(0,0,0);
text("hello",200,200,50,100);
rect(100, 200, 80, 100);
fill(255, 255, 255);
rect(50,200,30,50);
fill(201, 52, 201);
rect(50,250,10,55);
var n = 10;
for(var i = 0; i < n; i++){
    rect(120  , i * 10 + i * 5 , 10, 10);
}

fill(31, 148, 59);
for(var i = 0; i < n; i++){
    for(var j = 0; j < n; j++){
        rect(140 + (15 * j) , 15 * i, 10, 10);
    }
}
